crypto/fipsmodule/ec/p256_shared.h

```diff
@@ -51,7 +51,20 @@ typedef unsigned char P256_SCALAR_BYTES[33];
 
 static inline void p256_scalar_bytes_from_limbs(
     P256_SCALAR_BYTES bytes_out, const BN_ULONG limbs[P256_LIMBS]) {
+#ifdef RING_BIG_ENDIAN
+  for (int i = 0; i < P256_LIMBS; i++)
+  {
+#if BN_BITS2 == 64
+    BN_ULONG limb = CRYPTO_bswap8(limbs[i]);
+    OPENSSL_memcpy(bytes_out + i * 8, &limb, 8);
+#else
+    BN_ULONG limb = CRYPTO_bswap4(limbs[i]);
+    OPENSSL_memcpy(bytes_out + i * 4, &limb, 4);
+#endif
+  }
+#else
```

Comment:
In the recent merge from BoringSSL I brought in the following functions:

```
static inline crypto_word CRYPTO_load_word_le(const void *in) {
  crypto_word v;
  OPENSSL_memcpy(&v, in, sizeof(v));
  return v;
}

static inline void CRYPTO_store_word_le(void *out, crypto_word v) {
  OPENSSL_memcpy(out, &v, sizeof(v));
}
```
Please use these functions to eliminate the conditional compilation logic here.

Reply:
On re-testing I noticed that all this code is never even used on s390x these days - the whole file is only compiled in on x86_64 and aarch64 targets.  Therefore I've just reverted this change now.